xyz

```diff
@@ -86,14 +91,15 @@ while (($# > 0)) ; do
       node -p "require('$dir/package.json').version"
       exit
       ;;
-    -b|--branch)    branch="$1"           ; shift ;;
-    -e|--edit)      edit=true             ;;
-    -i|--increment) increment="$1"        ; shift ;;
-    -m|--message)   message_template="$1" ; shift ;;
-    -r|--repo)      repo="$1"             ; shift ;;
-    -s|--script)    scripts+=("$1")       ; shift ;;
-    -t|--tag)       tag_template="$1"     ; shift ;;
-    --dry-run)      dry_run=true          ;;
+    -b|--branch)         branch="$1"           ; shift ;;
+    -e|--edit)           edit=true             ;;
+    -i|--increment)      increment="$1"        ; shift ;;
+    -m|--message)        message_template="$1" ; shift ;;
+    --publish-command)   publish_command="$1"  ; shift ;;
```

Comment:
Let's indent this by three spaces to align it with the other `--` labels (and do the same for `--dry-run`).